eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/Imp/GovSpec.hs

```diff
@@ -486,22 +485,13 @@ proposalsWithVotingSpec =
         fmap (!! 3) getProposalsForest
           `shouldReturn` Node (SJust p116) []
     it "Proposals are stored in the expected order" $ do
-      modifyPParams $
-        ppMaxValSizeL .~ 1_000_000_000
+      modifyPParams $ ppMaxValSizeL .~ 1_000_000_000
       returnAddr <- registerRewardAccount
       deposit <- getsNES $ nesEsL . curPParamsEpochStateL . ppGovActionDepositL
       ens <- getEnactState
-      withdrawals <- do
-        creds <- arbitrary :: ImpTestM era (NonEmpty (StakeCredential (EraCrypto era)))
-        pairs <-
-          forM
-            creds
-            ( \cred -> do
-                Positive n <- arbitrary
-                ac <- getRewardAccountFor cred
-                pure (ac, Coin n)
-            )
-        pure $ Map.fromList (NE.toList pairs)
+      withdrawals <-
+        Map.fromList . map ((returnAddr,) . Coin . getPositive) . NE.toList
+          <$> (arbitrary :: ImpTestM era (NonEmpty (Positive Integer)))
```

Comment:
Since the test does not concern with the number of withdrawals in the `GovAction`, could we skip making multiple here and just have a `singleton`? I will update the PR, and await your approval or review comment 👍